task/build-vm-image/0.1/build-vm-image.yaml

```diff
@@ -148,6 +154,22 @@ spec:
         echo "$BUILD_DIR"
         ssh -v $SSH_ARGS "$SSH_HOST"  mkdir -p "$BUILD_DIR/workspaces" "$BUILD_DIR/scripts" "$BUILD_DIR/tmp" "$BUILD_DIR/tekton-results" "$BUILD_DIR/entitlement"
 
+        if [! -n "${CONFIG_TOML_FILE}" ]; then
+          echo "No CONFIG_TOML_FILE specified"
+          export CONFIG_TOML_FILE=config.toml
+          if [ -f /var/workdir/source/config.toml ]; then
+            echo "Using the config.toml file found in the repository root!"
+            echo "  Remove the config.toml file or set params.CONFIG_TOML_FILE to an invalid path to prevent its use."
```

Comment:
setting CONFIG_TOML_FILE to an invalid path is no longer an option

Reply:
You are right. They would have to create an empty config file and point to that.

Reply:
Or just remove the default config.toml (unless for whatever reason they want to keep it and make the build ignore it)